app/src/contexts/GameContext.js

```diff
@@ -25,18 +25,34 @@ export default function GameContextProvider({ children }) {
         });
     }
 
+    async function getCurrentBoard() {
```

Comment:
Itt definiálod a gettereket, de lentebb mégis közvetlenül exportálod a stateket, és a komponensekben is azokat használod. Akkor most melyik legyen?